src/main/java/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,30 @@
+package ladder.domain;
+
+public class LadderGame {
+
+    private final Participants participants;
+
+    private final Ladder ladder;
+    public LadderGame(Participants participants, Ladder ladder) {
```

Comment:
가독성을 위해 한줄 띄어놓는 것도 좋겠네요.
```suggestion
    private final Ladder ladder;
    
    public LadderGame(Participants participants, Ladder ladder) {
```

Reply:
반영하였습니다